docs/blog/2019-07-11-user-testing-accessible-client-routing/index.md

```diff
@@ -0,0 +1,193 @@
+
+Meetings are recommended for testing demos of a product with assistive technology, to co-design user experiences and conduct research, or to solve complex problems that customers have complained about. Tickets, on the other hand, are “like engaging a diverse team of QA analysts” to test usability and compatibility on important user journeys within your site. You can get video recordings of the sessions, which may come in handy for later analysis.
+
+In our meetings, we tested with:
+
+- Sam using NVDA and Chrome on Windows 10
+- Ka using JAWS and Firefox on Windows
+- Yvone using screen magnification in Chrome on a Samsung Note 9, and ZoomText for Chrome on Mac
+- Vu with Dragon NaturallySpeaking and the keyboard in Chrome
+- Carolyn with switch access
+
+In each of the sessions, we learned about the tester’s browsing method and in almost all cases was able to watch them browse through the prototypes. Asking a question to expand on a given detail provided lots of great insight (i.e. “tell me more about what makes that focus indicator confusing to you”). It helped to have the desired outcomes for the meetings defined and published ahead of time, so the testers knew what to keep in mind when testing and what questions they should try to answer.
+
+## Testing objectives: the techniques
+
+We wanted to know the best approach for guiding users with disabilities through JavaScript-heavy web applications, which don’t use traditional page reloads to reset the user’s focus point and announce new pages in assistive technologies.
+
+The [prototypes for these tests](https://marcysutton.com/prototype-testing-accessible-clientside-routing/#What-are-Prototypes) covered some of the common techniques for enhancing the page loading experience for users with disabilities, each centered around the mechanics of loading new content. These techniques included:
+
+- [Example 1](https://marcy.codes/prototypes/routing/example-1.html): No focus change, with a Live Region announcement for screen readers
+- [Example 2](https://marcy.codes/prototypes/routing/example-2.html): Focus reset to application wrapper element, with a Live Region announcement for screen readers
+- [Example 3](https://marcy.codes/prototypes/routing/example-3.html): Focus changed to a heading element in the newly changed content
+- [Example 4](https://marcy.codes/prototypes/routing/example-4.html): Focus changed to an element wrapping the newly changed content
+- [Example 5](https://marcy.codes/prototypes/routing/example-5.html): Focus changed to a wrapper element at 100% width, rather than side-by-side (added during testing session #3 on zooming/magnification)
+
+These prototypes used [Doggo Ipsum](https://doggoipsum.com/) for placeholder text, which made for some fun testing experiences. "Cuuuuuute long bois" was my personal favorite, captured in testing screenshots. :)
```

Comment:
I wanted to call attention to the use of Doggo Ipsum since the screenshots are sorta funny to visual readers...it did mention it in alt text for the image though. @amberleyromo is this _too_ obvious?

Reply:
Too obvious to call out specifically? No, I think not. I found it distracting when we were talking about the screenshots, so I think it's relieving to call out specifically.